spiketoolkit/validation/quality_metric_classes/utils/curationsortingextractor.py

```diff
@@ -236,9 +236,34 @@ def split_unit(self, unit_id, indices):
             self._roots.append(new_root_2)
 
             for feature_name in self.get_unit_spike_feature_names(unit_id):
+                if feature_name.endswith('_idxs'):
+                    continue
                 full_features = np.array(self.get_unit_spike_features(unit_id, feature_name))
```

Comment:
I wouldn't cast this as a numpy array since we don't want to load it all into memory.

Reply:
It was already cast as np.array(), I'll check if how can do it without that. 
Their is the same thing in merge function, I'll check for it too.

Reply:
you can check the type: if it's list, then you can cast it as np.array, otherwise you leave it as it is

Reply:
it's already numpy array ^^, so i remove np.array()

Reply:
And in fact their is nothing to change in merge funciton, my bad

Reply:
you can remove the np.array from the full_features and add:
```
if isinstance(full_features, list):
    full_features = np.array(full_features)
```
That should make sure all the indexing works properly

Reply:
Thanks :p
Done